CHANGES.md

```diff
@@ -1,3 +1,12 @@
+Release 5.29 (release date: 2024-05-07)
+======================================
+New Features
+------------------
+
+Other Changes
+------------------
+- Fixed bug related to dependency.
```

Comment:
This is not descriptive? Can you update this?

Reply:
Done